### ISSUE TITLE
0Ad==0.24 fails to load

### ISSUE THREAD

**Issue by igormorgado** [JustusAvramenko/delenda_est#47]
Actual repository fails to load in 0.24 0AD version with error:

![error](https://user-images.githubusercontent.com/2663394/120868096-95c6be00-c569-11eb-8af9-7ae9d47f61dc.png)

But using the commit version `b64180db002ff83c1ad7f0f2500cc5424a85cdff`  loads fine.

All other commits newer than `b64180db` fails with the same issue.

**Comment by igormorgado** [comment]
Ok. Game loads fine, but the match do not start. Log file attached.

[erros.txt](https://github.com/JustusAvramenko/delenda_est/files/6601512/erros.txt)

**Comment by igormorgado** [comment]
Just to let you know that works with SVN 0ad.

**Comment by hopeless-ponderer** [comment]
Hello @igormorgado,

I wrote the commits in question. I created a new feature, random civ selection groups, which only works with the A25 version of the game setup page.

I believe @JustusAvramenko is currently working toward compatibility with A25, even though the dependency declared in mod.json is still 0ad>=0.0.24. Perhaps the dependency should be updated to 0ad=0.0.25?

I apologize for the inconvenience.
